tooling/FST-1030-FSharp.Core nuget package netstandard 2.0 only.md

```diff
@@ -0,0 +1,59 @@
+# F# Tooling RFC FST-1030 - Make FSharp.Core nuget package netstandard2.0 only
+NOTE: This is a draft and remains to be reviewed by key contributors. Please contribute to the discussion and submit adjustments to the draft.
+
+The purpose of this RFC document is to develop a shared plan of record for the delivery of FSharp.Core in NuGet package form as a netstandard2.0 library.
+
+Discussion thread: https://github.com/fsharp/fslang-design/issues/482
+Suggestion thread: https://github.com/fsharp/fslang-suggestions/issues/893
+
+## Background
+Historically FSharp.Core.dll was compiled and delivered by Microsoft as a .NET 4.x binary installed under "Reference Assemblies". The DLL was signed and strong-named by Microsoft. Way, way back in time it was called fslib.dll.
+
+Subsequently, FSharp.Core.dll has been recompiled for "trimmed down" platforms: portable profiles, Xamarin profiles, .NET Core, .NET Standard and .NET Framework. There is even a version of FSharp.Core for the "Xamarin iOS TV" profile.... When provided by Microsoft, these were signed, strong-named and installed under "Reference Assemblies" on Windows. For other F# compilation environments like Xamarin, Mono, Cloud Sharper, FSharp.Formatting, Azure Notebooks, Azure Functions there was a bit of a mess (made considerably worse by the separation of sigdata/optdata files, now addressed)
+
+The F# Core Engineering Group created a NuGet package called FSharp.Core, partly to avoid a proliferation of "homebrew" packages appearing at that time.  For the last 3 years management and distribution of this nuget package has occured at the fsharp OSS site: https://github.com/dotnet/fsharp
+
+Today
+For the last 3 years the package has been released as a signed nuget package supporting both net45+ and netstandard2.0 TFMs.
+
+## implementation
+The implementation is easy.  Today at the fsharp oss project site: https://github.com/dotnet/fsharp, the fsharp.core nuget package and signed dll's are built.  We will remove the build for the desktop clr and all multi-fsharp.core testing support.
```

Comment:
```suggestion
The implementation is easy.  We will remove the build for the desktop clr and all multi-fsharp.core testing support. This will make the build of FSharp.Core emit a single assembly that targets `netstandard2.0`.
```